client/packages/reports/src/ReportsService.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+import { Routes, Route } from '@openmsupply-client/common';
+import { ListView } from './ListView';
+import { DetailView } from './DetailView';
+
+const ReportsService: FC = () => {
+  return (
+    <Routes>
+      <Route path="/:id" element={<DetailView />} />
+      <Route path="/" element={<ListView />} />
```

Comment:
Can't get breadcrumb to go back from reports since report is the top level link... should we add it in? Maybe on the icon or something? Not too sure...